### ISSUE TITLE
new raspberry pi 64bit OS !?

### ISSUE THREAD

**Issue by labdvm** [g4klx/MMDVMHost#738]
Hi, does somebody tested mmdvm-host on new raspberry pi 64bit bullseye OS ? 
Z32IT

**Comment by KE7FNS** [comment]
> Hi, does somebody tested mmdvm-host on new raspberry pi 64bit bullseye OS ? Z32IT

Yes, I've been running 64 Bit bullseye on a RPi 4 since bullseye 64-bit was announced for beta.  I built MMDVMHost with no issues.  I didn't encounter any issues with it running, but I also didn't run it for more than about an hour or so as I was just testing things.  

Theres a little bit of a dependency issue with a couple of libraries, but nothing too difficult to solve.  Since there is no longer an apt package for wiringpi use the github fork, and for ArduiPi_OLED you'll need to compile and install it, if you need it for OLED support.

Other than that its no different than 32-bit and the older Raspberry Pi OS's.  MMDVMHost uses a bit more ram than 32-bit though.

**Comment by labdvm** [comment]
Hi KE7FNS, I'm using mmdvm-host on RPi4/2G for dmr repeater with nextion display. I'm planning to migrate from RPiOS 32 to 64 bit and test it. Thanks.
Z32IT

**Comment by KE7FNS** [comment]
> Hi KE7FNS, I'm using mmdvm-host on RPi4/2G for dmr repeater with nextion display. I'm planning to migrate from RPiOS 32 to 64 bit and test it. Thanks. Z32IT

Are you currently using someone elses prebuilt image, like Pi-Star?

**Comment by labdvm** [comment]
Hi KE7FNS, I'm using pi-star on hotspot's only, on repeaters mmdvm-host.

**Comment by KE7FNS** [comment]
> Hi KE7FNS, I'm using pi-star on hotspot's only, on repeaters mmdvm-host.

I don't understand what "mmdvm-host" is.  Is it some prebuilt image that someone else created and you just wrote it to a SD card, or are you trying to setup all of the software from scratch.

**Comment by g4klx** [comment]
He said that he uses Pi-Star on his hotspots (only) and MMDVMHost on his repeaters. It's not difficult to understand, the name of this part of GitHub is MMDVMHost.

**Comment by KE7FNS** [comment]
> He said that he uses Pi-Star on his hotspots (only) and MMDVMHost on his repeaters. It's not difficult to understand, the name of this part of GitHub is MMDVMHost.

He clearly said "mmdvm-host".    MMDVMHost != mmdvm-host.

**Comment by g4klx** [comment]
Wow, so there's another piece of software out there called MMDVMHost/mmdvm-host? I never knew that!

Alternatively, you're just f*****g stupid.

**Comment by KE7FNS** [comment]
@g4klx Wtf is your problem?
